packages/verification-vue-component/src/style/components/verification/card/_base.scss

```diff
@@ -116,14 +116,18 @@
                 .verifier-image {
                     flex: 1 1 auto;
                     overflow: hidden;
-                    height: 60px;
                     padding-top: 5px;
 
                     img {
                         display: block;
                         width: auto;
                         max-width: 100%;
-                        height: 100%;
+                        height: 60px;
+
+                        &.wide {
```

Comment:
Why this wide class? There is no other `.verifier-image img` combination. Also keep in mind, this style needs to fit ALL verifier-images, we can't have multiple implementations for different image. WE define the format of the image needed, not the customer.

Reply:
Also, .wide is less wide than the default ;-)

Reply:
Adjusted as discussed. Adding transparent space to the switch logo was also needed.